myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :system do
+  before do
+    @user = create(:user)
+    @task1 = create(:task, name: 'Task 1', status: 'Not Started')
+    @task2 = create(:task, name: 'Task 2', status: 'In Progress')
+    @task3 = create(:task, name: 'Task 3', status: 'Done')
+  end
+
+  describe 'Task list page' do
+    it 'displays a list of tasks' do
+      visit tasks_path
+
+      expect(page).to have_content('Task 1')
+      expect(page).to have_content('Task 2')
+      expect(page).to have_content('Task 3')
+    end
+  end
+
+  describe 'Task creation' do
+    it 'creates a new task with valid input' do
+      visit new_task_path
+
+      fill_in 'Name', with: 'New Task'
+      select 'In Progress', from: 'Status'
+
+      click_button 'Create Task'
+
+      expect(page).to have_content('Task was successfully created.')
+      expect(page).to have_content('New Task')
+      expect(page).to have_content('In Progress')
```

Comment:
You should add a test for created data of the table of test db, also  should be updated or should be deleted

Reply:
Sure, I have add test for confirm created, updated or deleted task show correctly in task list.